desktop/libs/notebook/src/notebook/connectors/flink_sql.py

```diff
@@ -453,6 +462,20 @@ def _show_functions(self, database):
 
     return [{'name': function[0]} for function in function_list]
 
+  def _use_catalog(self, catalog):
+    session = self._get_session()
+    self.db.configure_session(session_handle=(session['id']), statement="USE CATALOG `%s`" % catalog)
+
+  def _use_database(self, catalog, database):
+    session = self._get_session()
+    if catalog and database:
```

Comment:
Nit: Can remove database in the check if we are assuming its present for the else part?